### PR TITLE
Update voice-tone.md

### LINKED ISSUE #3649: Style Guide update suggestions
# Some suggested improvements to the Style Guide

# General
The dotnet/docs repo has a style guide which contains a document called _**Voice and tone guidelines**_  (docs/styleguide/voice-tone.md ).
This is a great document and I am so glad to see these kinds of things. I wish all IT sites were as focused on writing quality!  

With that said, I'd like to add two additional guidelines:

1. **Avoid split infinitives** - While to boldly go may sound good on TV, split infinitives can reduce the readability. So, to go boldly...

2. **Avoid future tense**. Two points here - first is that in some non-English the concept of future tense is not the same as English making the docs harder to read for those folks. Secondly, when using the future tense, the obvious question is when. So if I say 'Learning PowerShell will be good for you" - the obvious question is when will it be good? To avoid that, just "Learning PowerShell is good for you". 

Also, the final two paragraphs could better be placed under a further guideline: **What Is it - So What" which could read:

You are writing documentation for technical professionals. You can assume that your readers have technical knowledge and understand the specific vocabulary for programming and IT Pro related terms.
Object-Oriented Programming, Class, Object, Function, Method, Cmdlet, Parameter, etc are all terms you can assume your reader understands. 

Whenever you introduce a new concept to the reader, for example describing the .NET Composite Formatting System - you should define the concept and only then explain the benefits and describe the usage. So instead of telling the reader that the Composite Formatting system is very useful, explain what it is first. The term/subject/concept should be defined first then explained. You can also explain a term in passing. So if you wanted to note that the Close() method is idempotent you could say:

>The Close() method is idempotent, meaning that you can call it multiple times and the effect is the same as if you called it once.

## Issues with Existing Topics
There are topics that currently violate these rules - those can be picked up as and when. As readers here may know, I've picked up a number of these. But having these three points in the style guide make those changes easier (ie I can 'update to reflect style guide').  :-)

### PR DESCRIPTION
**Pull request by doctordns**
# Updated voice document per #3649 

## Summary
1. Added some new tone guidelines as per issue 3649
2. Removed Future tense from existing text as a result.

Insert a short (one or two sentence) summary here.
Added a few common sense additions to the tone guide to help future authors.

Fixes #3649 

## Details

I am hoping that more PowerShell folks can join in the fun of adding to this documentation. I'd like to ensure that any documentation thus created is to a great standard. 

## Suggested Reviewers
@mairaw 
If you know who should review this, use '@' to request a review.

### CODE REVIEW

**Comment by doctordns** [comment]
With respect to this Pull Request - I note the warnings in the OpenPublishing.Build section. When I look at the details of he build report, I see LOADS of warnings. None of the warnings I can see relate to anything I've touched. Or though is that old curse ('just opening a source code file creates six other bugs in WIndows') from the days in Redmond come back to haunt! ;-)

File: styleguide/voice-tone.md [hunk 1]
```diff
@@ -70,9 +73,18 @@ they may not have the English vocabulary you have.
...
 >The Close() method is idempotent, meaning that you can call it multiple times and the effect is
 the same as if you called it once.
+
+### Avoid split infinitives
+While to boldly go may sound good on TV, split infinitives can reduce the readability of your content. So, to go boldly...
+
+### Avoid future tense
+In some non-English languages the concept of future tense is not the same as English. Using future tense can make your docuemnts harder to read. Additionally, when using the future tense, the obvious question is when. So if you say 'Learning PowerShell will be good for you" - the obvious question for the reader is when will it be good? Instead, just say "Learning PowerShell is good for you".
+
+### What is it - so what?
```
Comment (BillWagner) [inline styleguide/voice-tone.md]: This is a great point.  

@mairaw Should we highlight this more prominently?

**Review by BillWagner — APPROVED** [review]
Thanks for updating this @doctordns We appreciate it.
I've approved this, but I want to make sure @mairaw takes a look as well.

File: styleguide/voice-tone.md (hunk 1 above)
Comment (rpetrusha) [reply styleguide/voice-tone.md]: In terms of your questions about warnings, @doctordns, the warnings are not only from your PR, but from all the content published from the gitHub/dotnet repo; we're slowly working on addressing all of these issues. Ordinarily, the publishing process (the item that reads "OpenPublishing.Build" on the Conversions tab of the PR) will report whether your PR passed or succeeded with warnings, although it's not always accurate. It's still worthwhile to scan the actual report to see if any of the warnings might have been caused by your PR.

File: styleguide/voice-tone.md [hunk 2]
```diff
@@ -70,9 +73,18 @@ they may not have the English vocabulary you have.
 
 However, you are still writing for technical professionals. You can assume that your readers
 have programming knowledge and the specific vocabulary for programming terms. Object Oriented
-Programming, Class, and Object, Function and Method will all be familiar terms. However, not everyone
-reading your articles will have a formal computer science degree. Terms like 'idempotent' probably
+Programming, Class, and Object, Function and Method are all be familiar terms. However, not everyone
```
Comment (mairaw) [inline styleguide/voice-tone.md]: typo: are all be -> are all

**Review by mairaw — CHANGES_REQUESTED** [review]
I really like the changes. Left just a comment for a small typo to be fixed.

**Comment by doctordns** [comment]
I've made the update - thanks for the catch!

I think this can now be merged - but as ever, I could be wrong!

**Review by mairaw — APPROVED** [review]
You even found extra typos! Thanks @doctordns!

**Pull request merged**